webapp/package.json

```diff
@@ -21,7 +21,7 @@
     "@mui/material-nextjs": "^5.15.11",
     "@mui/x-tree-view": "^7.8.0",
     "@octokit/rest": "^20.0.2",
-    "next": "14.2.4",
+    "next": "14.2.10",
```

Comment:
I'm sorry mr bot, but if you bump here, you also have to bump eslint-config-next in the root.